api/grpc/health/v1/health.proto

```diff
@@ -25,24 +25,34 @@ option java_multiple_files = true;
 option java_outer_classname = "HealthProto";
 option java_package = "io.grpc.health.v1";
 
-message HealthCheckRequest {
+enum ServingStatus {
+  SERVING_STATUS_UNSPECIFIED = 0;
+  SERVING_STATUS_SERVING = 1;
+  SERVING_STATUS_NOT_SERVING = 2;
+  SERVING_STATUS_SERVICE_UNKNOWN = 3;  // Used only by the Watch method.
+}
+
+message CheckRequest {
+  string service = 1;
+}
+
+message CheckResponse {
+  ServingStatus status = 1;
+}
+
+message WatchRequest {
   string service = 1;
 }
 
-message HealthCheckResponse {
-  enum ServingStatus {
-    UNKNOWN = 0;
-    SERVING = 1;
-    NOT_SERVING = 2;
-    SERVICE_UNKNOWN = 3;  // Used only by the Watch method.
-  }
+message WatchResponse {
   ServingStatus status = 1;
 }
 
-service Health {
+
+service HealthService {
```

Comment:
I think this is a breaking change. Overall, I don't think we can or should change this file as it is not ours, but a shared API that any gRPC health client will rely on. 

Also, auraed doesn't actually use this file to create the endpoint; tonic-health has its own copy embedded into the create. This file is just for the macros.

Reply:
reverted